src/draw/nxp/vglite/lv_draw_vglite.c

```diff
@@ -257,6 +257,10 @@ static int32_t _vglite_evaluate(lv_draw_unit_t * u, lv_draw_task_t * t)
                     return 0;
                 }
 
+#if CHIPID == 0x255
```

Comment:
What is the `CHIPID`?

Reply:
Is the HW version of the GPU, defined in the vglite driver.

Reply:
Yes this CHIP ID is something defined in the kernel side driver, it gets included when the lvgl vglite API is enabled.